<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mailling</class>
 <widget class="QMainWindow" name="Mailling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mailing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendBtn_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);   border-radius: 10px;   padding: 20px;   text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }
</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uname_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="placeholderText">
     <string> À </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="placeholderText">
     <string>subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>571</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="placeholderText">
     <string>attachement here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="paswd_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>           Votre mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rcpt_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="placeholderText">
     <string> À </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>651</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);       text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }

</string>
    </property>
    <property name="placeholderText">
     <string>type your message here ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: rgb(98,70,137);   border-radius: 10px;   padding: 20px;   text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); }
</string>
    </property>
    <property name="text">
     <string>Browse ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-15</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="youssef.qrc">:/backgroundMailing.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_back3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {   font-family: 'Montserrat', sans-serif;   font-size: 10px;   color: #fff;   background-color: #023020;   border-radius: 10px;   padding: 20px;   text-shadow: 2px 2px #333;   box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3); 

background-color: rgb(205, 171, 247);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="youssef.qrc">
      <normaloff>:/arrow_391888.png</normaloff>:/arrow_391888.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>sendBtn_2</zorder>
   <zorder>uname_2</zorder>
   <zorder>subject_2</zorder>
   <zorder>file_2</zorder>
   <zorder>paswd_2</zorder>
   <zorder>rcpt_2</zorder>
   <zorder>msg_2</zorder>
   <zorder>browseBtn_2</zorder>
   <zorder>pb_back3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="youssef.qrc"/>
 </resources>
 <connections/>
</ui>
